src/api/app/jobs/consistency_check_job.rb

```diff
@@ -234,24 +234,24 @@ def dir_to_array(xmlhash)
     xmlhash.elements('entry') do |e|
       array << e['name']
     end
-    array.sort
+    array.sort!
   end
 
   def hash_diff(a, b)
     # ignore the order inside of the hash
-    (a.keys.sort | b.keys.sort).each_with_object({}) do |diff, k|
+    (a.keys.sort! | b.keys.sort!).each_with_object({}) do |diff, k|
```

Comment:
[nitpick] I wonder if this is what we really want here, or this: `(a.keys | b.keys).sort!`

Reply:
I agree.